test/SourceKit/SemanticTokens/semantic_tokens.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %sourcekitd-test -req=semantic-tokens %s -- %s | %FileCheck %s
+
```

Comment:
Do we get semantic tokens of buffers within a primary file?

Reply:
We do now. Added https://github.com/apple/swift/pull/69294/commits/1776e3753b4c82339662ffa210ff9a773a39ab53